commands/upgrade.go

```diff
@@ -0,0 +1,16 @@
+package commands
+
+import "github.com/spf13/cobra"
+
+var (
+	// UpgradeCommand is the command to upgrade things
+	UpgradeCommand = &cobra.Command{
+		Use:   "upgrade",
+		Short: "Upgrade clusters",
+		Long:  `Lets you upgrade a cluster`,
+	}
+)
```

Comment:
Those var block look weird in this situation.

```
// UpgradeCommand is the command to upgrade things.
var UpgradeCommand = &cobra.Command{
	Use:   "upgrade",
	Short: "Upgrade clusters",
	Long:  `Lets you upgrade a cluster`,
}
```

It is more concise, less indentation.

But not a blocker.

Also full stop is missing.

Reply:
If there was only one import, should I then also write import "foo" instead of a block? I thought Gophers prefer diffs to be minimal, hence all the fuzz about trailing commas in blocks like

```go
myslice := []string{
    s1,
    s22,
}
```

Reply:
> If there was only one import, should I then also write import "foo" instead of a block?

I don't think that improves readability, it also doesn't reduce indentation.

What I mean

```
var (
	someString = "abc"
	someInt    = 1
)
```

^ That improves readability. Also groups related vars together.

```
var (
	someStruct = type {
		a: 1,
	}
)
```

^ That doesn't improve readability.

Trailing commas are different story. They improve diffs.

Reply:
I don't think it's so important to spend more time on that. Do what you think is right.

Reply:
I'll leave it consistent with the rest of the command files then.